Yeti-experience/draft-song-yeti-testbed-experience.txt

```diff
@@ -425,13 +458,18 @@ Internet-Draft      Yeti DNS Testbed: Interim Results          June 2017
 
    o  the SERIAL when this information is active
 
+   XXX(that list describes the contents of files, not the files
+   themselves.  Can I see examples of the actual files that are
+   used?)XXX
+
    The operation is simple that each DM operator synchronize the files
-   with the information needed to produce the Yeti root zone.  When a
-   change is desired (such as adding a new server or rolling the ZSK), a
-   DM operator updates the local file and push to other DM.  A SOA
-   SERIAL in the future is chosen for when the changes become active.
+   with the information needed to produce the Yeti root zone.
+   XXX(how?)XXX When a change is desired XX(by whom?)XX (such as adding
+   a new server or rolling the ZSK), a DM operator updates the local
+   file and push to other DM XX(how?)XX.  A SOA SERIAL in the future is
+   chosen for when the changes become active.
```

Comment:
(Yeti-DM-Sync)[https://github.com/BII-Lab/Yeti-Project/blob/master/doc/Yeti-DM-Sync.md] introduce the Theory of Operation based on Git repositories. Paul also give a (presetnation on the git operation on Yeti DM)[http://yeti-dns.org/resource/2016workshop/1.Yeti%20Software.pdf]